FlowCal/plot.py

```diff
@@ -891,26 +894,49 @@ def hist1d(data_list,
             weights = None
 
         # Actually plot
-        if bins is not None:
-            n, edges, patches = plt.hist(y,
-                                         bins,
-                                         weights=weights,
-                                         normed=normed_area,
-                                         histtype=histtype,
-                                         edgecolor=edgecolor[i],
-                                         facecolor=facecolor[i],
-                                         **kwargs)
+        if packaging.version.parse(matplotlib.__version__) \
+                >= packaging.version.parse('2.2'):
```

Comment:
Is there a reason you chose to use the `density` keyword for `mpl 2.2` and above? Looks like it was introduced in 2.1: https://matplotlib.org/users/prev_whats_new/whats_new_2.1.0.html#density-kwarg-to-hist , so you could maybe more appropriately use it there too. This is a pretty narrow use case, though, and not terribly important.

Reply:
Two reasons:
- I saw no warning when running with matplotlib 2.1. If it doesn't complain, I'm not gonna fix it.
- More importantly, `plt.hist()` in matplotlib `2.1.0` does not interpret the `normed` or `density` arguments properly. More details here: https://github.com/matplotlib/matplotlib/issues/9557/ In the case of FlowCal, `plot.hist1d()` with `normed_area=True` does not produce the same plot with `matplotlib 2.1.0` as it does with any other version that I've tested, including `2.0`, `2.1.2`, and `2.2.x`. Again, this is due to an intrinsic problem with `plt.hist()`, therefore I think it's not something we can address without calculating the histogram ourselves. And I don't think we should do that for a matplotlib bug that has already been solved. Either way, what we do for matplotlib 2.1.0 doesn't matter because the result is wrong anyway.